### ISSUE TITLE
Accessibility -> Multiple form labels have been associated with the form controls

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#255]
## Description
**How to replicate:**
* Navigate to the A&R website on the test environment: [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/enter-data/frontcountry-camping?date=202210&parkName=TEST%20PARK&subAreaId=TEST&subAreaName=TEST%20SUBAREA&orcs=TEST)
* Log in and navigate to the “Enter Data” page
* Select a month, Park and a Sub-area, and click “Go”
* From the list of facilities, select one to add information using the “Edit” option. 
* On the resulting page check for the form labels. 
eg: 

![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/3acb5612-b513-4c98-8739-b3547f4fbd83)

**Expected Behavior:**
* The form controls on all the pages should have relevant form labels to be able to read by assistive technologies. One label should be associated with one form control. 

**Issues:**
* The form controls have been associated with multiple form labels. This issue can be seen on all form controls on the Enter data forms. 
eg: 

![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/7a7eb94d-c662-4f75-8634-6ca2ca85eac8)

![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/5ce38b03-8688-4365-b622-3633d400b80b)

## Linked issues

relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
